src/components/Router/index.js

```diff
@@ -231,11 +227,12 @@ class HistoryRouter extends Component {
       step: stepIndex,
       initialStep: stepIndex,
     }
-    this.unlisten = history.listen(this.onHistoryChange)
+    this.history = createHistory()
```

Comment:
I see `history` now lives within the Router component and there is an outstanding issue with history backtracking when there are 2 instances of the SDK running. 
Have you tried setting history with a unique ID as a solution? Something like this:
```
this.historyMap = { 
    id: uniqueId,
    history: createHistory() 
}
```
Note that this is just an idea & might not work. Or maybe you've already tried it.

Reply:
I thought about something like that, but I didn't try it. I'll give it a go

Reply:
Is it even possible to have multiple histories?

Reply:
The only problem we have now is with the back button. We have a ticket to fix that in our backlog so I will leave this as it's out of scope

Reply:
Added a link to these comments in the ticket